backend/Cargo.toml

```diff
@@ -29,12 +39,16 @@ pil-stark-prover = { git = "https://github.com/powdr-labs/pil-stark-prover.git",
 
 # TODO change this once Halo2 releases 0.3.1
```

Comment:
Don't we have to add `features = ["avx512"]` to `starky`, to be able to compile with `target-cpu=native`?

Reply:
Hmm, I see, it is activated via the `simd` feature.

It's just a bit weird that when running with `RUSTFLAGS='-Ctarget-cpu=native'`, it fails to compile if you don't activate the `simd` feature. But I don't see a better solution...

Reply:
It is activated via the simd feature, if you want for all backends, or just the starky one.

Yea also don't have a better alternative, it's the same for plonky3.